hack/tls/openssl.cnf

```diff
@@ -0,0 +1,31 @@
+[req]
```

Comment:
This can also be moved at top level along with other config files for user to modify as per the deployment needs.
BTW, why is the directory name "hack"?

Reply:
It's just a common practice in Go or K8s related project.

_"The [hack](https://github.com/kubernetes/kubernetes/blob/master/hack) directory contains many scripts that ensure continuous development of kubernetes, enhance the robustness of the code, improve development efficiency, etc. The explanations and descriptions of these scripts are helpful for contributors. For details, refer to the following guidelines."_

Source: [kubernetes/hack](https://github.com/kubernetes/kubernetes/tree/master/hack#overview)

Reply:
Thanks for the pointer! But going through it, it seems that's how Kubernetes has used the hack directory, but it is not a convention to be followed. Moreover, Kubernetes uses this directory to store all scripts related to building and testing for product stability and scripts for common updates -- for efficiency and uniformity.
For us, we already have a scripts directory in the root folder, having create-interfaces script to be used for deployment. Similar scripts, including the one to generate certificates, that are meant for deployment, can be placed in it.

Reply:
moved gen-certs under scripts dir.